### ISSUE TITLE
did you missed "influxdb_export.h"？

### ISSUE THREAD

**Issue by yf-web** [offa/influxdb-cxx#89]

**Comment by offa** [comment]
Can you describe the steps to reproduce the issue and your system (OS, compiler, version used) please?

**Comment by yf-web** [comment]
you can find 
`#include "influxdb_export.h"`
in "InfluxDBFactory.h".
but the InfluxDBFactory.h does not exist in the project.

**Comment by offa** [comment]
I assume you mean `influxdb_export.h` (not the factory) is not in the repository? The file is automatically generated by cmake, therefore not part of the repository.

**Comment by yf-web** [comment]
> I assume you mean `influxdb_export.h` (not the factory) is not in the repository? The file is automatically generated by cmake, therefore not part of the repository.

that's right!it's my fault,sorry!thank you so much! i think i need to learn how to use cmake.
